velox/common/memory/MemoryPool.cpp

```diff
@@ -408,11 +408,13 @@ MemoryPoolImpl::~MemoryPoolImpl() {
 
   if (isLeaf()) {
     if (usedReservationBytes_ > 0) {
+      LOG(ERROR) << "Bad memory usage track state : " << toString();
```

Comment:
How about?
```
Used memory leak: 
```

Reply:
Used this instead to make it consistent with the reserved memory leak message:
> Memory leak (Used memory):